src/components/Button/Button.md

```diff
@@ -4,6 +4,8 @@ While buttons can technically be used to navigate a user to another part of the
 
 Also, use concise, specific, self-explanatory text, usually a single word. They should include a verb. If acting on something, their text should include the noun they're acting on, e.g. *"Create Group"*.
 
+Use type='submit' for submitting forms.  Doing this will trigger the onClick action when the user presses the enter key within certain form fields.
```

Comment:
Wrap things that should look like code in backticks (e.g. `type="submit"`), so that they get rendered properly in Markdown.

Reply:
There's two spaces before `Doing`. (My OCD kills me.)

Reply:
Haha... old habits die hard.  I looked for some support from google, but all I found was strongly worded articles that say 2 spaces is definitely wrong.  I'll fix it. :-)